### PR TITLE
update console blueprints

### PR DESCRIPTION
**Pull request by yamini**
- add details page to synthetics blueprint card
- new gretel tuner card
- update gpt card and add details page

[Details in this doc](https://docs.google.com/document/d/1fkDFsw8c-BdqCHTfQ3jlHmZ8VhLIfaMKtLA3rMmmIak/edit)

### CODE REVIEW

File: use_cases/gretel.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
       "description": "Create highly accurate synthetic training data for ML models.",
       "cardType": "Console",
       "imageName": "synthetics.png",
-      "icon": "synthetics.png",
```
Comment (thesteady) [inline use_cases/gretel.json:9]: **important:** we need the icon here, as we are transitioning from using the `imageName` field to `icon`. I'm going to remove the `imageName` and card header images in the folder in another week or so.

File: use_cases/gretel.json [hunk 2]
```diff
@@ -290,6 +309,20 @@
         "trainingTime": "< 1 min",
         "bytes": 65300
       }
+    },
+    {
+      "gtmId": "use-case-image-synthetics",
```
Comment (thesteady) [inline use_cases/gretel.json:314]: **quick question (nonblocking):** i noticed this one is now below transform v1, which we previously pushed to the bottom, so just wanted to make sure the ordering was intentional :)

Comment (yamini) [reply use_cases/gretel.json:314]: Yes, we're not supporting the image synthetics app right now; the model's out of date. We're close to deprecating transform v1 but it does have utility for NER, so I figure we can keep it above image synthetics. It's not a huge deal though, it doesn't look like many users click through to the blueprints page.

File: use_cases/gretel.json (hunk 1 above)
Comment (yamini) [reply use_cases/gretel.json:9]: ah thanks, i didn't catch that. Good to know.

**Review by thesteady — APPROVED** [review]

**Pull request merged**